runtime/test/manifest-parser-test.js

```diff
@@ -86,6 +86,20 @@ describe('manifest parser', function() {
         description \`my store\`
       store Store1 of Person 'some-id' @7 in 'people.json'`);
   });
+  it('fails to parse an argument list that uses reserved words', () => {
+    try {
+      parse(`
+        particle MyParticle
+          in MyThing consume
```

Comment:
nit - you might want to test each of the `consume`, `provide`, `in`, `out` (also `inout`?) separately, because when all of them are in the same spec you aren't really testing each...

Reply:
Thanks, I had wanted to avoid creating a test per keyword but at the moment we are only checking two so I've opened a PR to split the test.

https://github.com/PolymerLabs/arcs/pull/1933